controllers/userController.js

```diff
@@ -57,4 +57,64 @@ const emailCheck = async (req, res) => {
   }
 };
 
-module.exports = { signUp, signIn, emailCheck };
+const myPageShow = async (req, res) => {
+  const email = req.user.email;
+  try {
+    const result = await userService.mypage(email);
+    return res.status(200).json({ data: result });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+
+const myAddress = async (req, res) => {
+  const email = req.user.email;
+  try {
+    const result = await userService.myAddressShow(email);
+    return res.status(200).json({ data: result });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+
+const addressPatch = async (req, res) => {
+  try {
+    await userService.myAddressPatch(req);
+    return res.status(200).json({ message: "addressPatch Success" });
```

Comment:
```suggestion
    return res.status(201).json({ message: "UPDATE_USER_ADDRESS_SUCCESS" });
```
수정에 적합한 status code를 사용해주세요. 메세지도 수정해주세요~

Reply:
네 확인했습니다 !